client_end_user/shared/src/commonMain/kotlin/presentation/auth/signup/registration/RegistrationScreenEffect.kt

```diff
@@ -1,7 +1,11 @@
 package presentation.auth.signup.registration
 
 sealed interface RegistrationScreenEffect {
-    data class NavigateToSubmitRegistrationScreen(val username: String, val password: String) :
-        RegistrationScreenEffect
+    data class NavigateToSubmitRegistrationScreen(
+        val username: String,
+        val email: String,
+        val password: String
+    ) : RegistrationScreenEffect
```

Comment:
why this data class has variables that are not related to its name ?
also the class name is more like a verb not a noun could make it more like a noun when adding "on" before it
"OnNavigateToSubmitRegistrationScreen"

Reply:
it's represent the effect itself. so it's okay to be a verb